### PR TITLE
Hack to mod tablet orientation when clicking context overlay

### PR DESCRIPTION
**Pull request by zfox23**
DO NOT TEST.

I kind of hate that I wrote this code, but I don't know another way to do this given the current tablet capabilities.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1892/HighFidelity-Beta-PR11033-223ed5b59a11a41890b89e234bb6015775e96513.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1892/HighFidelity-Beta-PR11033-223ed5b59a11a41890b89e234bb6015775e96513.dmg

File: interface/src/ui/overlays/ContextOverlayInterface.cpp [hunk 1]
```diff
@@ -33,6 +33,18 @@ ContextOverlayInterface::ContextOverlayInterface() {
 
     auto entityTreeRenderer = DependencyManager::get<EntityTreeRenderer>().data();
     connect(entityTreeRenderer, SIGNAL(mousePressOnEntity(const EntityItemID&, const PointerEvent&)), this, SLOT(createOrDestroyContextOverlay(const EntityItemID&, const PointerEvent&)));
+    connect(_tabletScriptingInterface->getTablet("com.highfidelity.interface.tablet.system"), &TabletProxy::tabletShownChanged, this, [&]() {
+        if (_contextOverlayJustClicked && _hmdScriptingInterface->isMounted()) {
+            QUuid tabletFrameID = _hmdScriptingInterface->getCurrentTabletFrameID();
+            QVariantMap props;
+            auto myAvatar = DependencyManager::get<AvatarManager>()->getMyAvatar();
+            glm::vec3 position = myAvatar->getJointPosition("Head") + 0.6f * (myAvatar->getOrientation() * Vectors::FRONT);
```
Comment (davidkelly) [inline interface/src/ui/overlays/ContextOverlayInterface.cpp]: So this is directly in front of your face, 60 cm away?  Do we not want it off to a side slightly, offset down/up, etc...?  Just asking -- I really have no idea where it "should" go.  It sorta feels like this is right where you are looking when you click the (i) but maybe that isn't correct.  Just a comment, fwiw...

Comment (zfox23) [inline interface/src/ui/overlays/ContextOverlayInterface.cpp]: Updated, good feedback.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1905/HighFidelity-Beta-PR11033-c22798a8e64adfbd0682b5be512b8c06a5b7dd25.exe

File: interface/src/ui/overlays/ContextOverlayInterface.cpp [hunk 2]
```diff
@@ -33,6 +33,17 @@ ContextOverlayInterface::ContextOverlayInterface() {
 
     auto entityTreeRenderer = DependencyManager::get<EntityTreeRenderer>().data();
     connect(entityTreeRenderer, SIGNAL(mousePressOnEntity(const EntityItemID&, const PointerEvent&)), this, SLOT(createOrDestroyContextOverlay(const EntityItemID&, const PointerEvent&)));
+    connect(_tabletScriptingInterface->getTablet("com.highfidelity.interface.tablet.system"), &TabletProxy::tabletShownChanged, this, [&]() {
+        if (_contextOverlayJustClicked && _hmdScriptingInterface->isMounted()) {
+            QUuid tabletFrameID = _hmdScriptingInterface->getCurrentTabletFrameID();
+            QVariantMap props;
+            auto myAvatar = DependencyManager::get<AvatarManager>()->getMyAvatar();
+            props.insert("position", vec3toVariant(myAvatar->getEyePosition() + glm::quat(glm::radians(glm::vec3(0.0f, 30.0f, 0.0f))) * (0.65f * (myAvatar->getHeadOrientation() * Vectors::FRONT))));
```
Comment (davidkelly) [inline interface/src/ui/overlays/ContextOverlayInterface.cpp]: 👍

**Pull request merged**